### ISSUE TITLE
"avg_cost" why accumulated by batch_loss/total_batch, instead of batch_loss/batch_size ?

### ISSUE THREAD

**Issue by Harryhaha** [aymericdamien/TensorFlow-Examples#81]
with tf.Session() as sess:
    sess.run(init)
    # Training cycle
    for epoch in range(training_epochs):
        avg_cost = 0.
        total_batch = int(mnist.train.num_examples/batch_size)
        # Loop over all batches
        for i in range(total_batch):
            batch_xs, batch_ys = mnist.train.next_batch(batch_size)
            # Fit training using batch data
            _, **c** = sess.run([optimizer, cost], feed_dict={x: batch_xs,
                                                          y: batch_ys})
            **# Compute average loss
            avg_cost += c / _total_batch_**
        # Display logs per epoch step
        if (epoch+1) % display_step == 0:
            print "Epoch:", '%04d' % (epoch+1), "cost=", "{:.9f}".format(avg_cost)

When accumulate "avg_cost", why added by **c/total_batch**, instead of **c/batch_size** ?
Since I understand "avg_cost" is the average cost for each instance.

Thanks !!!
